src/views/poi_bloc/Wiki.jsx

```diff
@@ -0,0 +1,25 @@
+/* global _ */
+import React from 'react';
+import ExtendedString from '../../libs/string';
+
+const WikiBlock = ({
+  block,
+}) => {
+  return <div className="poi_panel__info__wiki">
+    { block.description &&
+      <p className="poi_panel__description__ellipsis">
+        { block.description }
```

Comment:
I noticed you removed `htmlEncode` form `block.description`. I don't think it's useful there, otherwise if the string can contain HTML meant to be interpreted, [dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) will be required.